packages/components/src/composite/stories/index.story.tsx

```diff
@@ -11,35 +11,31 @@ import { isRTL } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import {
-	Composite,
-	CompositeGroup,
-	CompositeRow,
-	CompositeItem,
-	useCompositeStore,
-} from '..';
+import * as Composite from '..';
 import { UseCompositeStorePlaceholder, transform } from './utils';
 
 const meta: Meta< typeof UseCompositeStorePlaceholder > = {
-	title: 'Components/Composite (V2)',
+	title: 'Components/Composite',
 	component: UseCompositeStorePlaceholder,
 	subcomponents: {
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		Composite,
+		Root: Composite.Root,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeGroup,
+		Group: Composite.Group,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeRow,
+		Row: Composite.Row,
 		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
-		CompositeItem,
+		Item: Composite.Item,
```

Comment:
There's another inconvenience of the dot notation. It would be nice if subcomponents were directly inferred.

Reply:
I can try to use object spread + a utility to filter out non-components stuff, something like 	`getSubComponents( Composite )`

Reply:
Definitely looks like overkill. Let's go with the manual one.